xpano/gui/pano_gui.h

```diff
@@ -55,4 +58,4 @@ class PanoGui {
   std::future<algorithm::StitchingResult> pano_future_;
 };
 
-}  // namespace xpano::gui
+}  // namespace xpano::gui
```

Comment:
nit: please don't change the whitespace if not needed, this screws up the history a bit